packages/Liquid-Tests.package/LQPollDraftTests.class/instance/setUp.st

```diff
@@ -0,0 +1,5 @@
+running
+setUp
+
+	super setUp.
+	self pollDraft: LiquidPollDraft new
```

Comment:
[sand] Same as above. Are instance variables the way to go?

Reply:
My comment above also applies here.